src/library/scala/collection/Map.scala

```diff
@@ -288,11 +288,18 @@ trait MapOps[K, +V, +CC[_, _] <: IterableOps[_, AnyConstr, _], +C]
   override def mkString(sep: String): String = super.mkString(sep)
   override def mkString: String = super.mkString
 
-  @deprecated("Consider requiring an immutable Map or fall back to Map.concat", "2.13.0")
-  def + [V1 >: V](kv: (K, V1)): CC[K, V1] = mapFactory.from(new View.Appended(toIterable, kv))
+  @deprecated("Consider requiring an immutable Map or fall back to Map.concat.", "2.13.0")
+  def + [V1 >: V](kv: (K, V1)): CC[K, V1] =
+    mapFactory.from(new View.Appended(toIterable, kv))
 
   @deprecated("Use ++ with an explicit collection argument instead of + with varargs", "2.13.0")
-  def + [V1 >: V](elem1: (K, V1), elem2: (K, V1), elems: (K, V1)*): CC[K, V1] = mapFactory.from(new View.Concat(new View.Appended(new View.Appended(toIterable, elem1), elem2), elems))
+  def + [V1 >: V](elem1: (K, V1), elem2: (K, V1), elems: (K, V1)*): CC[K, V1] =
+    mapFactory.from(new View.Concat(new View.Appended(new View.Appended(toIterable, elem1), elem2), elems))
+
+  @deprecated("Consider requiring an immutable Map.", "2.13.0")
+  def -- [V1 >: V](that: IterableOnce[K]): CC[K, V1] = {
```

Comment:
The type parameter `V1` is not needed, and we should return a `C`. (You can see in `immutable.MapOps` the right signature to use)